Swift/BRCrypto/BRCryptoNetwork.swift

```diff
@@ -292,3 +311,60 @@ public final class NetworkFee: Equatable {
         return CRYPTO_TRUE == cryptoNetworkFeeEqual (lhs.core, rhs.core)
     }
 }
+
+///
+/// A NetworkPeer is a Peer on a Network.  This is optionally used in Peer-to-Peer modes to
+/// specfify one or more peers to connect to for network synchronization.  Normally the P2P
+/// protocoly dynamically discovers peers and thus NetworkPeer is not commonly used.
+///
+public final class NetworkPeer: Equatable {
+    // The Core representation
+    internal let core: BRCryptoPeer
+
+    /// The network
+    public let network: Network
+
+    /// The address
+    public let address: String
+
+    /// The port
+    public let port: UInt16
+
+    /// The public key
+    public let publicKey: String?
+
+//    /// The serialization to allow reconstructing this peer
+//    public var serialization: Data {
+//        var bytesCount: Int = 0
+//        let bytes = cryptoPeerSerialize(core, &bytesCount)
+//        return Data (bytes: bytes!, count: bytesCount)
+//    }
+
+    internal init (core: BRCryptoPeer, take: Bool) {
+        self.core = (take ? cryptoPeerTake (core) : core)
+        self.network = Network (core: cryptoPeerGetNetwork (core), take: false)
+        self.address = asUTF8String (cryptoPeerGetAddress (core))
+        self.port    = cryptoPeerGetPort (core)
+        self.publicKey = cryptoPeerGetPublicKey (core)
+            .map { asUTF8String ($0) }
+    }
+
+    internal convenience init? (network: BRCryptoNetwork, address: String, port: UInt16, publicKey: String?) {
+        guard let core = cryptoPeerCreate (network, address, port, publicKey)
+            else { return nil }
+
+        self.init (core: core, take: false)
+    }
+    
+//    internal convenience init? (network: BRCryptoNetwork, serialization: Data) {
+//        var bytes = [UInt8](serialization)
+//        guard let core = cryptoPeerCreateFromSerialization (network, &bytes, bytes.count)
+//            else { return nil }
+//
+//        self.init (core: core, take: false)
+//    }
+
+    public static func == (lhs: NetworkPeer, rhs: NetworkPeer) -> Bool {
```

Comment:
Do we need an `IsIdentical()` at the C level?

Reply:
I don't know that we do (for Java I did the same as you); just a question.

Reply:
Yes, I'll update.

Reply:
Perhaps needs a Java update, for consistency.

Reply:
Yes, pushing shortly.